misc/license_headers.py

```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env python
-# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
-
```

Comment:
`license_header` configure itself to not have a header, should I revert the deletion ? should I change the header to `BUSL-1.1` ?

Reply:
I am not sure I understand the question. :thinking: Do you mean that you remove it because the file sets its own header or something like that? 
In any case, I think it is probably better/clearer to set it to `BUSL-1.1`